src/apps/store/controllers/book.controller.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  JsonController,
+  Get,
+  Post,
+  Param,
+  QueryParams,
+  Put,
+  Body,
+  Authorized,
+} from 'routing-controllers';
+import { Service } from 'typedi';
+import { BookCreateDto } from '@apps/store/dtos/book/create.dto';
+import { BookUpdateDto } from '@apps/store/dtos/book/update.dto';
+import { BookService } from '@apps/store/services/book.service';
+import { PaginateType, UserRole } from '@shared/type';
+
+@Service()
+@JsonController('/book')
+export class BookController {
+  constructor(private readonly bookService: BookService) {}
+
+  @Get('/all')
+  @Authorized(UserRole.MANAGER)
```

Comment:
All of user types can view the book list, therefore pls remove the permission checking.